source/agora/test/Base.d

```diff
@@ -892,7 +892,7 @@ public class TestAPIManager
 
         The name registry is a full node that exposes a more advanced interface.
         It is not part of the `nodes` array, but is reachable as a designed
-        address (currently "name.registry").
+        address (currently "10.8.8.8").
```

Comment:
https://github.com/bosagora/agora/pull/2900/commits/f4436f3863929908a9ee83d39fa2c8f0ef1ca836 was setting it to `8.8.8.8`

Reply:
Yep, but I figured using the private IP range would be safer, *just in case*